docs/rules/block-spacing.md

```diff
@@ -1,8 +1,8 @@
-# Disallow or enforce spaces inside of single line blocks (block-spacing)
+# Disallow or enforce spaces between an open block token and the next token on the same line (block-spacing)
```

Comment:
It would be good to mention close block token as well, if that's possible without making this too verbose.